tests/OperationProcessor.spec.ts

```diff
@@ -233,11 +250,273 @@ describe('OperationProcessor', async () => {
     expect(publicKey2!.owner).toBeDefined();
     expect(publicKey2!.owner!).toEqual('did:sidetree:updateid' + (numberOfUpdates - 1));
 
-    const deleteOperationBuffer = await OperationGenerator.generateDeleteOperation(didUniqueSuffix, '#key1', privateKey);
```

Comment:
Design sanity: how can the test without `await operationProcessor.processBatch([createOp!]);` being called? Are each subsequent test dependent on the previous test? I hope not.